spec/system/admin/bulk_order_management_spec.rb

```diff
@@ -473,33 +473,33 @@
           visit_bulk_order_management
         end
 
-        xit "allows filters to be used in combination" do
-          pending "#9811"
+        it "allows filters to be used in combination",
+           retry: 2, retry_wait: 2 do
```

Comment:
Why the wait? Did you see any changes in success here?

Reply:
I can't say I did - I did not investigate whether it played a role, for this spec. I've removed this option now.

Thanks!